src/furniture/search.js

```diff
@@ -0,0 +1,32 @@
+import callService  from '../utils/services/call.js'
+import normalizeFurnitureApiResult  from './common/normalize-furniture-api-result.js'
+
+export default function searchFurniture (query, options) {
+
+  // API
+  options = options || {}
+  var limit = options.limit || 50
+  //var offset = options.offset || 0
```

Comment:
Not working on this right now, but I created https://github.com/archilogic-com/services/issues/251 to remind us to make this happen when we redesign the APIs or even add it in an earlier milestone (which also means a tiny change in dashboard & furniture app)